Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -403,8 +403,12 @@ extension ActivityStreamPanel {
         let siteBGColor = topSiteItemCell.contentView.backgroundColor
 
         let site = self.topSitesManager.content[indexPath.item]
+        let domainURL = NSURL(string: site.url)?.domainURL().absoluteString ?? site.url
```

Comment:
I'll change it to simply drop the entire path. But keep everything else.
We have a few simple UI tests that test Top Sites. But they just go to home pages. So they don't have this issue.